lib/src/main/java/com/orange/ods/compose/component/control/OdsSlider.kt

```diff
@@ -226,7 +219,6 @@ fun OdsSliderLockups(
                 valueRange = valueRange,
                 steps = steps,
                 onValueChangeFinished = onValueChangeFinished,
-                colors = OdsSliderDefaults.colors(activeTickColor = OdsTheme.colors.surface.copy(alpha = ActiveTickColorAlpha))
```

Comment:
Should we apply the same change as `OdsSlider`?

```kotlin
colors = SliderDefaults.colors(activeTickColor = OdsTheme.colors.surface.copy(alpha = ActiveTickColorAlpha))
```

Reply:
No, when I try it changes the appearance of the active tick color in light theme and this is not what we want.